source/utilities.tex

```diff
@@ -18990,7 +18990,7 @@
 \begin{example}
 \begin{codeblock}
 using namespace std;
-vector<int> v = ...
+vector<int> v = ...;
```

Comment:
This isn't valid syntax. Could you perhaps use `vector<int> v = @\commentellip@;`?